### ISSUE TITLE
How "training loss" ""test loss" and "test metric" are defined?

### ISSUE THREAD

**Issue by qjiao1** [lululxvi/deep-learning-for-indentation#8]
Would you please explain how the "training loss" ""test loss" and "test metric" are defined?
I'm not sure why there are two values associated with training/test loss.
![image](https://user-images.githubusercontent.com/23302486/135331347-0e107a22-bd06-4cdd-b99f-3835317106fd.png)

**Comment by lululxvi** [comment]
"loss" is the loss you defined to train the network. Two values means there are more than one loss terms. Not sure what code you used, but it seems that the second value is the regularization loss.

Metric is the metirc you defined in your code.
